registry-port-forward.sh

```diff
@@ -17,4 +17,22 @@ run() {
     kubectl port-forward -n kube-system svc/docker-registry 30666:5000
 }
 
-run
+run_launchd() {
+  label="k8s-registry-port-forward"
+
+  launchctl remove "$label" 2> /dev/null || true
+
+  launchctl submit \
+    -l "$label" \
+    -p "$(which kubectl)" \
+    -- kubectl --kubeconfig="$PWD/files/kubeconfig.yaml" port-forward -n kube-system svc/docker-registry 30666:5000
```

Comment:
The `$KUBECONFIG` variable should be available, if not we should fix it.

You can run `make -C example port-forward` to relaunch the port forward process.

If it still not working, I would check the differences between GNU make and the one on macOS. If the `.ONESHELL` directive does not work, we might need to tweak the Makefile.